examples/viewer.py

```diff
@@ -937,14 +939,29 @@ def next_frame() -> None:
         action="store_true",
         help="Override configured lighting to use synthetic lighting for the stage.",
     )
-
+    
     args = parser.parse_args()
 
     # Setting up sim_settings
     sim_settings: Dict[str, Any] = default_sim_settings
-    sim_settings["scene"] = args.scene
-    sim_settings["scene_dataset_config_file"] = args.dataset
+
+    # this interactive viewer was developed with these arguments,
+    # so make them the default if none provided
+    if args.scene == "NONE":
+        scene_name = "./data/test_assets/scenes/simple_room.glb"
```

Comment:
Why dont you use `./data/test_assets/scenes/simple_room.glb` as the value of `default` value for `--scene` in the argument parser?

Reply:
ah, I didn't see that was an option here until now 😅:
parser.add_argument(
        "--scene",
        default="NONE",
        type=str,
        help='scene/stage file to load (default: "NONE")',
    )